pkgs/dxvk-nvapi/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  pkg-config,
+  vulkan-headers,
+  ninja,
+  meson,
+  windows,
+  pins,
+}: let
+  inherit (pins) dxvk-nvapi;
+in
+  # TODO: Add dxvk-nvapi-vkreflex-layer
+  stdenv.mkDerivation {
+    pname = "dxvk-nvapi";
+    inherit (dxvk-nvapi) version;
+
+    src = dxvk-nvapi;
+
+    postPatch = ''
+      # Use Vulkan Headers from nixpkgs
+      substituteInPlace meson.build \
+        --replace-fail "./external/Vulkan-Headers/include" "${vulkan-headers}/include"
+    '';
+    strictDeps = true;
+    nativeBuildInputs = [
+      pkg-config
+      meson
+      ninja
+    ];
+    buildInputs = lib.optional stdenv.hostPlatform.isWindows windows.pthreads;
+    mesonBuildType = "release";
+    doCheck = true;
+    __structuredAttrs = true;
```

Comment:
oh, what's this?

Reply:
This is to make sure it uses pthreads to build, same as dxvk_2 does upstream